src/Root.tsx

```diff
@@ -0,0 +1,23 @@
+import { Route, HashRouter as Router, Routes, Navigate } from 'react-router-dom';
+import { App } from './App';
+import { NotFound } from './components/NotFound';
+import { Home } from './components/Home';
+import { PeoplePage } from './components/PeoplePage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path='/' element={<App/>}>
+          {/* Redirect from '/home' to '/' */}
+          <Route path="/home" element={<Navigate to="/" replace />} />
+
+          <Route index element={<Home/>} />
+          <Route path='/people' element={<PeoplePage/>} />
+          <Route path="/people/:personName" element={<PeoplePage />} />
```

Comment:
There is a potential redundancy here. Both routes `/people` and `/people/:personName` are handled by the `PeoplePage` component. If `PeoplePage` is not designed to differentiate between these two routes, it might lead to confusion or unexpected behavior. Consider whether you need both routes and if `PeoplePage` should handle them differently.